integration-tests/deployment/ccip/deploy_feed_chain.go

```diff
@@ -0,0 +1,58 @@
+package ccipdeployment
+
+import (
+	"fmt"
+	"math/big"
+
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/smartcontractkit/chainlink-common/pkg/logger"
+	"github.com/smartcontractkit/chainlink/integration-tests/deployment"
+	"github.com/smartcontractkit/chainlink/v2/core/gethwrappers/generated/aggregator_v3_interface"
+	"github.com/smartcontractkit/chainlink/v2/core/gethwrappers/generated/mock_v3_aggregator_contract"
+)
+
+const (
+	LINK     = "LINK"
+	WETH     = "WETH"
+	DECIMALS = 18
+)
+
+var (
+	LINK_PRICE = big.NewInt(5e18)
+)
+
+func DeployFeeds(lggr logger.Logger, chain deployment.Chain) (deployment.AddressBook, map[string]common.Address, error) {
+	ab := deployment.NewMemoryAddressBook()
+	//TODO: Maybe append LINK to the contract name
+	linkTV := deployment.NewTypeAndVersion(PriceFeed, deployment.Version1_0_0)
```

Comment:
hm this should be whatever typeAndVersion is actually in the contract - does the mock aggregator not have one?

Reply:
mock aggregator doesn't have one. That's why I added the TODO to make it unique with different aggregators later.